internal/data/payments.go

```diff
@@ -264,6 +264,39 @@ func (p *PaymentModel) GetAll(ctx context.Context, queryParams *QueryParams, sql
 	return payments, nil
 }
 
+func (p *PaymentModel) GetAllReadyToPatchAnchorTransactions(ctx context.Context) ([]Payment, error) {
```

Comment:
Same here about naming, this is not valid for all ready-to-patch, but only the ones that are ready to patch for completion, right?